openml/extensions/sklearn/extension.py

```diff
@@ -602,6 +634,60 @@ def _check_multiple_occurence_of_component_in_flow(
                 known_sub_components.add(visitee.name)
                 to_visit_stack.extend(visitee.components.values())
 
+    def _extract_sklearn_parameter_docstring(self, model):
+        def match_format(s):
+            return "{}\n{}\n".format(s, len(s) * '-')
+        s = inspect.getdoc(model)
+        s1 = "Parameters"
+        s2 = ["Attributes", "See also", "Note", "References"]
+        try:
+            index1 = s.index(match_format(s1))
+        except ValueError as e:
```

Comment:
Is this triggered in practice?

Reply:
Not really. However, since this design is based completely on another repo's design, I thought it would be more robust to account for a failure nonetheless.
I'm certain though I am not covering all possible breaking points.